.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!--  Thanks for sending a pull request!
+Please describe accurately what this PR does, and why we need it.
+Please make sure to point to whatever issues it fixes.
```

Comment:
I would drop this to have less noise in the text box. Seeing only the three headlines, there is a chance I will read them and answer them instead of just submitting an empty description.

Reply:
Done.